### PR TITLE
Functional tests - Fix some random errors

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix Design > Add new theme and Orders > Status tab tests 
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI and Nightly are :green_circle: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/6544684638
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/BO/orders/index.ts [hunk 1]
```diff
@@ -431,7 +424,7 @@ class Order extends BOBasePage {
    * @param row {number} Order row on table
    * @returns {Promise<string|null>}
    */
-  downloadInvoice(page: Page, row: number): Promise<string|null> {
+  downloadInvoice(page: Page, row: number): Promise<string | null> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/orders/index.ts]:
```suggestion
  async downloadInvoice(page: Page, row: number): Promise<string | null> {
```

File: tests/UI/pages/BO/orders/index.ts [hunk 2]
```diff
@@ -441,7 +434,7 @@ class Order extends BOBasePage {
    * @param row {number} Order row on table
    * @returns {Promise<string|null>}
    */
-  downloadDeliverySlip(page: Page, row: number): Promise<string|null> {
+  downloadDeliverySlip(page: Page, row: number): Promise<string | null> {
```
Comment (Progi1984) [inline tests/UI/pages/BO/orders/index.ts]:
```suggestion
  async downloadDeliverySlip(page: Page, row: number): Promise<string | null> {
```

**Review by Progi1984 — COMMENTED** [review]
Two small comments

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**